package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase-sample/prime-sdk-ts",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
is this a breaking change? should it be 0.4.0?

Reply:
see web3 answer